code/modules/clothing/under/accessories/permits.dm

```diff
@@ -1,40 +1,40 @@
 //This'll be used for gun permits, such as for heads of staff, antags, and bartenders
```

Comment:
File path was changed too. Make sure our file gets moved if it hadn't already.

Reply:
Ours is under /fluffstuff so I don't think it needs to be moved
https://github.com/VOREStation/VOREStation/blob/master/code/modules/vore/fluffstuff/custom_permits_vr.dm